### ISSUE TITLE
AVT 1 - Text Input  has DAP violation when disabled

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4542]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

## Detailed Description
Go to the React Component and select the disable knob
Run DAP and the following violations are present (see screenshot):
<img width="933" alt="Screen Shot 2019-11-01 at 4 33 33 PM" src="https://user-images.githubusercontent.com/21676914/68058040-dbecd900-fcc5-11e9-9b27-adf38bede9f5.png">

**Comment by dakahn** [comment]
@aagonzales is the label text meant to take on this low contrast when disabled as well? Seems like that's where we're getting this violation.

**Comment by snidersd** [comment]
@dakahn This issue is a DAP false positive and it has been reported in the [DAP repo issue# 1271](https://github.ibm.com/IBMa/Tools-Rules-HTML/issues/1271)

**Comment by dakahn** [comment]
As discussed offline disabled components are exempt from 1.4.3's contrast requirements. Looks like an error with DAP 👍
